sgt-back-end/server.ts

```diff
@@ -92,18 +92,34 @@ app.get('/api/grades', async (_, res, next) => {
 app.post('/api/grades', async (req, res, next) => {
   try {
     // SQL query to get all rows from the grades table
+    const { name, course, score } = req.body;
+
+    if (!name) {
+      throw new ClientError(400, 'no name');
+    }
+    if (!course) {
+      throw new ClientError(400, 'no course');
+    }
+    if (!score) {
+      throw new ClientError(400, 'no score');
+    }
+
     const sql = `
-    INSERT INTO "grades" ("name" , "course" , "score" )
-      VALUES ('Tobin Bell' , 'Killing' , '200')
-      RETURNING *
+    UPDATE "grade"
```

Comment:
The POST needs to be an Insert statement, not an Update. The original INSERT you had was correct, you just needed to change from `VALUES ('Tobin Bell' , 'Killing' , '200')` to `VALUES ($1, $2, $3)`.